charts/kong/templates/migrations.yaml

```diff
@@ -55,6 +55,8 @@ spec:
       - name: {{ template "kong.name" . }}-migrations
         image: {{ include "kong.getRepoTag" .Values.image }}
         imagePullPolicy: {{ .Values.image.pullPolicy }}
+        securityContext:
+        {{ toYaml .Values.containerSecurityContext | nindent 10 }} 
```

Comment:
It occurred to me that we should be doing conditional value setting here, basically don't set this field as empty if the `values.yaml` is empty... @rainest?

Reply:
`toYaml` (which is [basically `yaml.Marshal()`](https://github.com/helm/helm/blob/a499b4b179307c267bdf3ec49b880e3dbd2a5591/pkg/engine/funcs.go#L79-L90) apparently handles nil inputs fine, and returns an empty object, which is valid. Actual template output is:

```
      containers:
      - name: ingress-controller
        securityContext:
      
          {}  
```
Which is a bit ugly, but valid. :shrug: